### PR TITLE
Skurz initial openliberty

### PR DESCRIPTION
**Pull request by uberskigeek**
### Checklist:

- [ ] Read the [Code of Conduct](https://github.com/appsody/website/blob/master/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/appsody/website/blob/master/CONTRIBUTING.md).

- [ ] Followed the [commit message guidelines](https://github.com/appsody/website/blob/master/CONTRIBUTING.md#commit-message-guidelines).

- [ ] Stack adheres to [Appsody stack structure](https://github.com/appsody/website/blob/master/content/docs/stacks/stack-structure.md).

### Modifying an existing stack:

- [ ] Updated the stack version in `stack.yaml`

### Contributing a new stack:

- Describe how application dependencies are managed:

- Explain how Appsody file watcher is utilized:

- Describe other Appsody environment variables defined by the stack image:

- Describe any limitations and known issues:

### CODE REVIEW

File: incubator/java-openliberty/image/project/Dockerfile [hunk 1]
```diff
@@ -6,6 +6,13 @@ RUN apt-get update && \
 
 # Copy and build the dev.appsody:java-openliberty parent pom
 COPY ./pom.xml /project/pom.xml
+
+# make usr dir for server definitions
```
Comment (scottkurz) [inline incubator/java-openliberty/image/project/Dockerfile]: What are these for?

File: incubator/java-openliberty/image/project/Dockerfile [hunk 2]
```diff
@@ -26,10 +33,15 @@ RUN cd /project/user-app/target/liberty/wlp/usr/servers && \
     mv defaultServer/* /config/
 
 # Step 2: Package Open Liberty image
-FROM open-liberty:kernel-java8-openj9
+FROM open-liberty:{{.stack.liberty_version}}-kernel-java8-openj9
 
 COPY --chown=1001:0 --from=0 /config/ /config/
 COPY --chown=1001:0 --from=0 /project/user-app/target/*.war /config/apps
 
+# make usr dir for server definitions
```
Comment (scottkurz) [inline incubator/java-openliberty/image/project/Dockerfile]: Same question:  what are these for?

File: incubator/java-openliberty/image/Dockerfile-stack [hunk 3]
```diff
@@ -14,6 +14,7 @@ RUN  /project/util/check_version build
 
 WORKDIR /project/
 RUN mkdir -p /mvn/repository
+#RUN mvn -B -Dmaven.repo.local=/mvn/repository -N io.takari:maven:wrapper -Dmaven=3.6.2
```
Comment (scottkurz) [inline incubator/java-openliberty/image/Dockerfile-stack]: we can remove

File: incubator/java-openliberty/image/project/pom.xml [hunk 4]
```diff
@@ -24,7 +24,7 @@
         <!-- OpenLiberty runtime -->
         <liberty.groupId>io.openliberty</liberty.groupId>
         <liberty.artifactId>openliberty-runtime</liberty.artifactId>
-        <version.openliberty-runtime>19.0.0.12</version.openliberty-runtime>
+        <version.openliberty-runtime>{{.stack.liberty_version}}</version.openliberty-runtime>
         <version.liberty-maven-plugin>3.1</version.liberty-maven-plugin>
```
Comment (scottkurz) [inline incubator/java-openliberty/image/project/pom.xml:28]: That's good.. what about also:
1. The two usages in the 'enforcer' plugin:

>                         <regex>19.0.0.12</regex>
>                            <regexMessage>OpenLiberty runtime version must be 19.0.0.12</regexMessage>

 2.  A rephrasing of the README.md doc:  
`This stack is based on OpenJDK with container-optimizations in OpenJ9 and `Open Liberty v19.0.0.12`

File: incubator/java-openliberty/templates/default/pom.xml [hunk 5]
```diff
@@ -95,6 +95,12 @@
                 <groupId>io.openliberty.tools</groupId>
                 <artifactId>liberty-maven-plugin</artifactId>
                 <configuration>
```
Comment (scottkurz) [inline incubator/java-openliberty/templates/default/pom.xml]: Why add this if we don't need to ?   We don't need to right?

**Review by scottkurz — CHANGES_REQUESTED** [review]

File: incubator/java-openliberty/image/project/pom.xml (hunk 4 above)
Comment (uberskigeek) [reply incubator/java-openliberty/image/project/pom.xml:28]: Missed those.  Will add.

File: incubator/java-openliberty/image/Dockerfile-stack (hunk 3 above)
Comment (uberskigeek) [reply incubator/java-openliberty/image/Dockerfile-stack]: ok

File: incubator/java-openliberty/image/project/Dockerfile (hunk 1 above)
Comment (uberskigeek) [reply incubator/java-openliberty/image/project/Dockerfile]: I think these were there to attempt to put the usr dir someplace we could access it. I think Adam fixed this by opening permissions on the directory where liberty is installed. I'll remove these

**Pull request closed without merging**